server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -102,16 +101,27 @@ public static enum OpenSearchRequestType {
     private final Path extensionsPath;
     final List<DiscoveryExtension> extensionsList;
     List<DiscoveryExtension> extensionsInitializedList;
+    Map<String, DiscoveryExtension> extensionIdMap;
+    Map<String, List<String>> extensionRestActionsMap;
     TransportService transportService;
     ClusterService clusterService;
     ExtensionNamedWriteableRegistry namedWriteableRegistry;
 
+    /**
+     * Instantiate a new ExtensionsOrchestrator object to handle requests and responses from extensions.
+     *
+     * @param settings  Settings from the node the orchestrator is running on.
+     * @param extensionsPath  Path to a directory containing extensions.
+     * @throws IOException  If the extensions discovery file is not properly retrieved.
+     */
     public ExtensionsOrchestrator(Settings settings, Path extensionsPath) throws IOException {
         logger.info("ExtensionsOrchestrator initialized");
         this.extensionsPath = extensionsPath;
         this.transportService = null;
         this.extensionsList = new ArrayList<DiscoveryExtension>();
         this.extensionsInitializedList = new ArrayList<DiscoveryExtension>();
+        this.extensionIdMap = new HashMap<String, DiscoveryExtension>();
```

Comment:
Can we add a comment here mentioning how this map will be used by Rest Handler?

Reply:
Honestly I don't know yet. Per comment on #69 it might be deleted.  My initial intent was as a place to point an API request to the correct extension (IP and Port) but that may end up being handled by the work I do in #69.

If necessary to get this PR approved I can delete this.  Or I can just continue to iterate on this PR as I work on #74.